docker/tap-schema/1015_dp01_dc2_tap_schema.sql

```diff
@@ -0,0 +1,1185 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('dp01_dc2_catalogs', null, 'Data Preview 0 includes five tables based on the DESC’S Data Challenge 2 simulation of 300 square degrees of the wide-fast-deep LSST survey region after 5 years. All tables contain objects detected in coadded images.', 0);
+INSERT INTO tap_schema.tables11 (schema_name, table_name, table_type, utype, description, table_index) VALUES ('dp01_dc2_catalogs', 'dp01_dc2_catalogs.position', 'table', null, 'Select astrometry-related parameters for objects detected in the coadded images, such as coordinates, footprints, patch/tract information, and deblending parameters. (29 columns)', 0);
+INSERT INTO tap_schema.columns11 (table_name, column_name, datatype, arraysize, xtype, description, utype, unit, ucd, indexed, principal, std, column_index) VALUES ('dp01_dc2_catalogs.position', 'objectId', 'long', null, null, 'Unique id.', null, null, null, 0, 0, 0, null);
+INSERT INTO tap_schema.columns11 (table_name, column_name, datatype, arraysize, xtype, description, utype, unit, ucd, indexed, principal, std, column_index) VALUES ('dp01_dc2_catalogs.position', 'coord_ra', 'double', null, null, 'RA-coordinate', null, '', 'pos.eq.ra', 0, 0, 0, null);
```

Comment:
For future reference, but not an obstacle today: I would like the (ra,dec) column pair that was actually used for the Qserv sharding to have UCDs `pos.eq.ra;meta.main` / `pos.eq.dec;meta.main`, especially if there are any other alternate ra/dec columns in the table.  (In this case there aren't, so it's not critical.)

@fritzm and I should talk about whether we should set `indexed=1` for these as well; that's debatable.  Again, not an obstacle to this PR.